sky/shell/platform/android/io/flutter/view/FlutterMain.java

```diff
@@ -246,4 +260,16 @@ private static void initNative(Context applicationContext) {
             throw new RuntimeException(e);
         }
     }
+
+    private static void initAot(Context applicationContext) {
+        File aotSnapshot = new File(applicationContext.getApplicationInfo().nativeLibraryDir,
+                                    System.mapLibraryName("aot_snapshot"));
```

Comment:
The script snapshot is called `snapshot_blob.bin`.  If we want to be symmetric with that name, we might want to call this `snapshot_aot` rather than `aot_snapshot`.